.github/workflows/build_shelly_homekit.yaml

```diff
@@ -0,0 +1,25 @@
+on:
+  push:
+  pull_request:
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
we have docker containers for mos - repository name is `mgos/mos`, could we use that?

Reply:
I've given it a go, however, I've run into a problem: a local build requires docker – which, from what I gather, would require running docker in a docker container. In this context, what improvements does using docker image hold over ubuntu with mos ppa?

Reply:
I agree with @ZeevoX, his current way is the better one.